torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -78,6 +127,14 @@ struct ConstantPropagator {
           (*new_output)->setType(n->outputs()[i]->type());
         }
         n->outputs()[i]->replaceAllUsesWith(*new_output);
+      } else if (outputs[i].isTuple() && n->outputs().size() == 1) {
```

Comment:
Why does the tuple have to be the only output?

Reply:
In general it's a bug for an op to not have one output. It should be either None, T, or a Tuple. So this should never be hit. But i included it because it would be weird if one of the outputs of an op was constantified, and we still kept the op in the graph. and ran it again